src/status_im2/contexts/chat/messages/content/link_preview/view.cljs

```diff
@@ -22,6 +22,6 @@
           {:title           title
            :description     description
            :link            hostname
-           :thumbnail       (:url thumbnail)
+           :thumbnail       (or (:url thumbnail) url)
```

Comment:
You can use one function instead of the macro `or` plus the function call to get the `:url`:

```clojure
(get thumbnail :url url)
```

Reply:
Done